tools/dataProtector/dataProtectorCore/processProtectedData.md

```diff
@@ -18,10 +18,12 @@ const taskid = await dataProtector.processProtectedData({
   maxPrice: 10,
   args: 'arg1 arg2',
   inputFiles: ['https://example.com/file1', 'https://example.com/file2'],
-  secrets: {
-    1: 'secret1',
-    2: 'secret2',
-  },
+  <!-- prettier-ignore-start -->
+  secrets: { // [!code focus]
+    1: 'secret1', // [!code focus]
+    2: 'secret2', // [!code focus]
+  }, // [!code focus]
+  <!-- prettier-ignore-end -->
```

Comment:
There should be no code highlights here as this is the global usage of the function, all parameters should be visible and readable. (No blur)
I'll change that 👍